packages/cozy-client/src/utils.js

```diff
@@ -41,3 +41,14 @@ export const isQueryLoading = col => {
 export const hasQueryBeenLoaded = col => {
   return col.lastFetch
 }
+
+/**
+ * Returns whether a query has more data to fetch
+ */
+export const hasQueryMoreToFetch = col => {
+  if (!col) {
+    console.warn('hasQueryMoreToFetch called on falsy value.') // eslint-disable-line no-console
+    return false
+  }
+  return col.fetchStatus === 'loaded' && col.hasMore
```

Comment:
Why is there a check on the fetchStatus here ?

Reply:
Because if there is more data to fetch, `hasMore` is `false` while `fetchStatus` is not `loaded` (so `pending` or `loading`).

This function is used to fetch more data after the first batch (whose number of results depends on `limitBy`), so I thought we need the first result before fetching more.

But you're right it's not necessary

Reply:
But in that case… is this function really necessary? Checking `result.hasMore` seems easier to me than `hasQueryMoreToFetch(result)` + the import that goes with it.

Reply:
It's the same approach as for `hasQueryBeenLoaded` I think it brings a better readability of the code, isn't it ?

Reply:
I'm going to save everyone a little time by shutting down that PR. If it hasn't been done so far, it's probably for the reasons you mention :)